aries-site/src/pages/components/textinput.mdx

```diff
@@ -45,7 +45,7 @@ A TextInput's visual state informs the user of its ability to be interacted with
 
 ### Password
 
-Used to maintain the privacy of the password a user has entered. This is achieved by applying type="password" to the TextInput.
+Used to maintain the privacy of the password, PIN or any credentials allowing access to an account a user has entered. This is achieved by applying type="password" to the TextInput.
```

Comment:
```suggestion
Used to maintain the privacy of the password, PIN, or any credentials allowing access to an account a user has entered. This is achieved by applying type="password" to the TextInput.
```
Minor comment but we should add a comma here. See brand guidance: https://brandcentral.hpe.com/brand-central/content/training-standards

Reply:
@jcfilben  thank you! looks great.